ci-operator/step-registry/openshift/e2e/gcp/cilium/default/openshift-e2e-gcp-cilium-default-workflow.yaml

```
@@ -0,0 +1,16 @@
+workflow:
+  as: openshift-e2e-gcp-cilium-default
```

```suggestion
  as: openshift-e2e-gcp-cilium
```
I'd rather remove `default` and add `openshift-e2e-gcp-cilium-someotherconfig` later

---

I was just doing this for consistency, but now your version makes more sense, as there there is `network-conf-disable-cno` instead of `cilium-conf`.